content/_index.md

```diff
@@ -0,0 +1,24 @@
+# Linux as Firmware
+**Reinventing the wheel by implementing drivers for firmware again and again.**</br>
+**Not with LinuxBoot!**
+
+![](images/linuxboot_info.png)
+
+### Platform init and memory training
```

Comment:
I think this section is a bit deep for the landing page. I suggest more of a why this is a good thing type of discussion, and maybe links to FAQ for this stuff?

Reply:
Done